service/discovery/azure/compute.go

```diff
@@ -724,9 +618,179 @@ func (d *azureComputeDiscovery) keyURL(diskEncryptionSetID string) (string, *arm
 	return util.Deref(keyURL), &kv.DiskEncryptionSet, nil
 }
 
+// getSiteProperties returns properties for WebApp und functions
+func (d *azureComputeDiscovery) getSiteProperties(site *armappservice.Site) (ni []voc.ResourceID, publicAccess bool, resourceLogging *voc.ResourceLogging) {
+
+	ni = []voc.ResourceID{}
+	resourceLogging = &voc.ResourceLogging{}
+
+	// If a mandatory field is empty, the whole function is empty
+	if site == nil {
+		return
+	}
+
+	// Get virtual network subnet ID
+	if site.Properties.VirtualNetworkSubnetID != nil {
+		ni = []voc.ResourceID{voc.ResourceID(util.Deref(site.Properties.VirtualNetworkSubnetID))}
+	}
+
+	// Check if resource is public available
+	if util.Deref(site.Properties.PublicNetworkAccess) == "Enabled" {
```

Comment:
I am not sure if it is the case here as well. But mostly, the Azure SDK defines Types and constants with underlying type string that holds this (magic) string.

Reply:
That is right, but in that case I couldn't find the corresponding constant.